### ISSUE TITLE
An Idea for On-boarding New Developers with a Video Tutorial Walk Through of a PR

### ISSUE THREAD

**Issue by iarobinson** [withbetterco/guidebook#8]
Hey John,

I was thinking it would be useful for me to make a screencast that might be useful for the company.

My idea is that I record myself for the day on Github. I can talk through all my decision making process:
- Read and understand an issue assigned to me
- Create a new git branch
- Change the code
- Add the Pull Request to the GitHub
- Assign the team leader the issue

This could be added to the WithBetter YouTube channel, or we could just add this to a private repository.

Just an idea. Let me know if you think it's worth pursuing. :)

**Comment by Johnsalzarulo** [comment]
@iarobinson - I love this idea - I would love for you to do something like this - 

Here's my couple points of direction: 

- First - we need to make sure it's a client project and an issue that isn't sensitive to their code and development process being semi-public. Medoplex or WhizTutor are good candidates, Whiz Tutor or an internal project is the best one as I don't have to get anyone's permission to do something like this. Otheriwse we'd have to obscure the name. We have some clients that we carry NDA's with. 

- Second - we are making a couple tweaks to the project management process very soon. Here's a quick overview of the changes coming: 
   - We are gong to start using [Milestones](https://github.com/withbetterco/medoplex/milestone/2) and [Project boards](https://github.com/withbetterco/medoplex/projects/3) consistently so everyone knows priorities. - So, that's how you'll know what's up next and each developer will be responsible to keep their issues updated on the board. 
   - Then - when a PR is opened it's assigned for review 
   - BUT - the original issue the developer started remains open and assigned to the original developer until the PR associated gets merged. 
   - Bottom line - individual developers are going to be more responsible to own features all the way through to deployment. Most times, that means hounding Manuel or I to get a review closed out and done. 
   - I'm still getting this stuff caught up and defined but this should be implemented in the next week or two at most. 

So - let's keep these two pending changes in mind when doing this project. 

I love it - I'm more than happy to pay some extra hours around the work to make it happen, help however I can. I'm open to how you want to release it and title it as well. 

I think it could be interesting just to do a "Day in the life of a remote developer" and explain your process, thinking and then just like time-lapse the dev work. hahaha.

**Comment by Johnsalzarulo** [comment]
Oh one other tweak to our process is that all PR's will require screenshots when they make visual changes.
